src/main/java/yablunovskyi/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,42 @@
+package yablunovskyi.bookstore.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import yablunovskyi.bookstore.dto.BookDto;
+import yablunovskyi.bookstore.dto.CreateBookRequestDto;
+import yablunovskyi.bookstore.service.BookService;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/books")
+public class BookController {
+    private final BookService bookService;
+    
+    @GetMapping
+    public List<BookDto> findAll() {
```

Comment:
Correct me if I'm wrong, but we should use find when we want to return Optional(on repository layer) and get on other layers